test/functional/ui/mouse_spec.lua

```diff
@@ -153,7 +153,8 @@ describe('Mouse input', function()
     end)
 
     it('in tabline to the left moves tab left', function()
-      if os.getenv("TRAVIS") and helpers.os_name() == "osx" then
+      if os.getenv("TRAVIS") and (helpers.os_name() == "osx"
```

Comment:
How about having a general way to disable flaky checks and then have .travis.yml export an environment variable under these conditions to cause the flaky tests to be disabled?  That would also allow me to disable the flaky tests on some of the more touchy Debian buildds.